cql3/restrictions/statement_restrictions.hh

```diff
@@ -455,7 +455,7 @@ public:
     bool pk_restrictions_need_filtering() const;
 
     bool ck_restrictions_need_filtering() const {
-        if (_clustering_columns_restrictions->empty()) {
+        if (expr::is_empty_restriction(_new_clustering_columns_restrictions)) {
```

Comment:
A more neutral name for is_empty_restriction is is_true(). But let's ignore that now.